README.adoc

```diff
@@ -85,12 +85,11 @@ We provide a few unsupported scripts that can help to get started:
 
 * You can use the link:hack/crc-openshift.sh[`crc-openshift.sh`] script to create an OpenShift cluster on your local machine.
 * If you are familiar to minikube, you may try the link:hack/k8s-minikube.sh[`k8s-minikube.sh`] script. It has the option to install Dex which is useful if you want to test with OpenID.
+* You can also use link:https://kind.sigs.k8s.io/[kind], use link:hack/setup-kind-in-ci.sh[`start-kind.sh`] script to create a single node KinD cluster with metallb enabled for testing Kiali against a real environment.
```

Comment:
Two small corrections:
- The start-kind.sh link is wrong. It should be `link:hack/start-kind.sh`
- `use`word is used twice in the same sentence, maybe you can use a similar word like `run start-kind.sh script`